distrobox-create

```diff
@@ -166,9 +166,20 @@ generate_command() {
 
 # First, check if the image exists in the host
 if ! podman image exists "${container_image}"; then
-	echo "Image not found, run this command first:"
-	echo "	podman pull ${container_image}"
-	exit 1
+    echo >&2 "Image not found."
+    echo >&2 "Do you want to pull the image now?[y/n]"
+    read -r yn
+    case $yn in
+        [Yy]* )
+            podman pull ${container_image}
+            break
+            ;;
+        [Nn]* )
+            echo >&2 "next time, run this command first:"
+            echo >&2 "  podman pull ${container_image}"
+            exit 1
+            ;;
```

Comment:
to remove the shellcheck warning about the brake at line 175, you have to remove it from there and add a default case for it:

```
	*) # Default case: If no more options then break out of the loop.
		break ;;
```

this way we also solve the default to yes

Reply:
The warning still remains, i'll just add a warning message and stop it there.